linux_os/guide/system/selinux/selinux_policytype/oval/shared.xml

```diff
@@ -3,7 +3,7 @@
     <metadata>
       <title>Enable SELinux</title>
       <affected family="unix">
-        <platform>multi_platform_rhel</platform>
+        <platform>multi_platform_all</platform>
```

Comment:
@mildass this should be `multi_platform_rhel` and `multi_platform_fedora`. Not all platforms will implement SELinux.

Reply:
@redhatrises ok, thanks. I wasn't sure about it, if rhel+fedora or multi_platform_all is correct.